config/logging.yaml

```diff
@@ -0,0 +1,46 @@
+# enable logging output inline in rails console
+:console_inline: true
+# colorize log including level token based on severity
+:colorize: true
+# adds an extra line after each log message with information from which file:line and method
+# was this message logged
+:log_trace: false
+#do not log certain paths (example  /notices/get_new)
+:ignored_paths: []
+# configuration of all loggers used in app
+:loggers:
+  # this logger is parent of all other loggers, by default this is the only one having appender
+  # which is used by all other loggers
+  :root:
+    # only messages with severity >= this level from all loggers will be logged
+    :level: debug
```

Comment:
All my concerns are resolved, except this one. I'd appreciate list of values which are allowed here.